docs/sites.yml

```diff
@@ -2078,3 +2078,14 @@
   built_by: Mill3 Studio
   built_by_url: https://mill3.studio/en/
   featured: false
+  - title: Bestvideogame.deals
```

Comment:
Hey @Koen87, can you remove two spaces from the beginning of the `title` line so the `-` is the first character, please? It's currently invalid YAML. Thanks!

Reply:
Removed!